content/millennium/r4/bulk-data/group-export.md

```diff
@@ -0,0 +1,305 @@
+* When the `_type` parameter is not provided, the resources to be exported will be determined by the scopes provided in the authorization token.
+* When the `_typeFilter` parameter is provided, the `_type` parameter MUST be provided as well.
+* `_typeFilter` does not support queries for the following resources: `Location`, `Organization`, `Patient`, `Provenance`
+* `_typeFilter` does not support queries with the following search parameters: `_count`, `_contained`, `_containedType`, `_elements`, `_id`, `_include`, `_lastUpdated`, `_revinclude`, `_sort`, `_summary`, `_total`, `encounter`, `identifier`, `patient`, `subject`
+* Supplying `_typeFilter` with the same resource multiple times is not supported.
+* `includeAssociatedData` valid parameters are `LatestProvenanceResources` and `RelevantProvenanceResources`
+* `_outputFormat` valid parameters are `application/fhir+ndjson`, `application/ndjson`, `ndjson`
+* `_outputFormat` defaults to `application/fhir+ndjson` when not provided.
+* The following parameters are not supported: `_elements`, `patient`
+
+_Provenance Behavior_
+
+* When the `_type` parameter is not provided and the token includes the Provenance read scope, all relevant Provenance resources associated with each of the non-provenance resources will be exported by default. You MAY use the  `includeAssociatedData` parameter to return only the latest provenance.
+* When the `_type` parameter is provided with specific resources and includes 'Provenance', all relevant Provenance resources associated with each of the non-provenance resources will be exported by default.
+* When both the `includeAssociatedData` parameter and the `_type` parameter are provided, the `_type` parameter MUST include `Provenance`
+* Provenances for `Location`, `Organization`, and `Practitioner` will not be exported.
+
+### Headers
+
+<%= headers head: {'Accept': 'application/fhir+json', 'Prefer': 'respond-async', Authorization: '&lt;OAuth2 Bearer Token>'} %>
+
+Optional Header
+
+<%= headers head: {'Prefer': 'handling=lenient'} %>
+
+When the Prefer: handling=lenient header is provided any unknown or unsupported parameters will be ignored as specified [here](http://hl7.org/fhir/uv/bulkdata/STU2/export.html#query-parameters).
+
+### Example
+
+Request
+
+    GET https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Group/11ec-d16a-c763b73e-98e8-a31715e6a2bf/$export
+
+Response
+
+    Status: 202 Accepted
+    Content-Location: https://fhir-ehr.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/bulk-export/jobs/<job_id>
+
+<%= disclaimer %>
+
+### Errors
+
+The common [errors] and [OperationOutcomes] may be returned.
+
+## Status Request
+
+Use the URL returned in the `Content-Location` header from the kick-off request to retrieve the status of a bulk data export:
+
+    GET /bulk-data/jobs/<job_id>
```

Comment:
should we avoid mentioning the path at all here and other similar places and just say
**GET <URL from Content-Location header>**

that way if we happen to change the path we don't have to modify the documentation as  this path is not standard and its our own thing

Reply:
@fpivonka , @MadhurRaj thoughts on this? 
For now I have updated this to bulk-exports instead of data: https://github.com/cerner/fhir.cerner.com/pull/899/commits/45938f3e974ba246b2abf5f2e3b74fab37db9838#diff-6963b85421ccad81bf2c26b848074843817dfa021595650e04f7ff89e1e85e31R149

Reply:
@ebbp123  I'm ok with whatever the group decides. I like having a concrete example but I'm not as technical as others so just having <URL from Content-Location header> may be fine.

Reply:
Checked with Madhur and he agrees with Seada, so I have updated the documentation: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR133